src/main/java/org/asamk/signal/commands/SendCommand.java

```diff
@@ -55,7 +55,9 @@ public void attachToSubparser(final Subparser subparser) {
         mut.addArgument("--message-from-stdin")
                 .action(Arguments.storeTrue())
                 .help("Read the message from standard input.");
-        subparser.addArgument("-a", "--attachment").nargs("*").help("Add file as attachment");
+        subparser.addArgument("-a", "--attachment").nargs("*").help("Add file as attachment."
+                + "Data URI encoded attachments can be added and must follow the RFC 2397. Additionally a file name can be added, e.g. "
+                + "data:<MIME-TYPE>;filename=<FILENAME>;base64,<BASE64 ENCODED DATA>.");
```

Comment:
```suggestion
        subparser.addArgument("-a", "--attachment").nargs("*").help("Add an attachment. "
                + "Can be either a file path or a data URI. Data URI encoded attachments must follow the RFC 2397. Additionally a file name can be added, e.g. "
                + "data:<MIME-TYPE>;filename=<FILENAME>;base64,<BASE64 ENCODED DATA>.");
```
Also it would be good to update the man page in `man/signal-cli.1.adoc`